specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1305,7 +1305,7 @@ at deployment time or at run time.
 
 RMI-IIOP is defined by chapters 5, 6, 13, 15,
 and section 10.6.2 of the CORBA 2.3.1 specification, available at
-_http://www.omg.org/cgi-bin/doc?formal/99-10-07_ , and by the Jakarta™
+_http://www.omg.org/cgi-bin/doc?formal/99-10-07_ , and by the Jakarta
```

Comment:
This should be left as "Java(TM) Language..".  This is a reference to the Java to IDL Mapping tool...